style the homepage, remove carousel and antd

shanseet opened the pull request:
Updated app logo from Rachel with the letters saved as svg instead of text

shanseet committed:
Reformat homepage, fix sidenav when removing antd

shanseet committed:
remove antd from dependency list

shanseet committed:
fix console error

shanseet committed:
update applogo svg and add welcome text

zhuhanming commented:
Looks awesome, just some general feedback:

Can we standardise the navbar font? It's quite jarring when Montserrat is used as a "subfont" of Open Sans, and not the other way round. I think pure Open Sans is good enough

<img width="246" alt="Screenshot 2020-07-07 at 12 09 39 AM" src="https://user-images.githubusercontent.com/45617494/86615018-72f0fa00-bfe6-11ea-89cf-7c3624bdda24.png">

There's some issue with the menu bar not responding upon click. Do see clip below:

![ezgif com-optimize-2](https://user-images.githubusercontent.com/45617494/86615320-e2ff8000-bfe6-11ea-922b-739ef4282e60.gif)

I think some parts can also be bolded. For example, asking me how many options I want to have - the question text can be bolded accordingly.

Lastly, I see the `makeStyles` and the `classes.paper` are still being used in the app. What's going to be the way forward with this? Should we extract it out to some global level styling? If not, lots of variations will appear again as time passes.

shanseet committed:
Merge branch 'master' of https://github.com/auturec/app2 into shanon/style-homepage

shanseet committed:
combine listitem and link using component prop

shanseet committed:
Merge branch 'master' of https://github.com/auturec/app2 into shanon/style-homepage

shanseet committed:
change default font to open sans

shanseet commented:
Changed the fonts to Open Sans, right now only added Montserrat to the homepage character slides' titles.
Also fixed the menu tabs, they should be working fine.

As to extracting styles and deciding between makestyles and scss, will work on it in a different branch!
